packages/abstractions/tests/authentication/test_api_key_authentication_provider.py

```diff
@@ -59,3 +59,17 @@ async def test_header_location_authentication(mock_request_information):
     await provider.authenticate_request(mock_request_information)
     assert "api_key" in mock_request_information.request_headers
     assert mock_request_information.headers.get("api_key") == {"test_key_string"}
+
+
+def test_https_prefix_in_allowed_host():
```

Comment:
Any chance you can update the value in line 5 as well to remove the prefix to fix the tests failing?